ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/local-users.j2

```diff
@@ -32,5 +32,10 @@ no {{ local_user_cli }}
 {%             set local_ssh_key_cli = local_ssh_key_cli ~ " ssh-key " ~ local_user.ssh_key %}
 {{ local_ssh_key_cli }}
 {%         endif %}
+{%         if local_user.secondary_ssh_key is arista.avd.defined %}
```

Comment:
I think we need to nest this `if` under the regular ssh key `if` above. Otherwise someone could provide the secondary_ssh_key alone and it would generate invalid CLI.

Reply:
Fixed